src/test/java/step3/CarTest.java

```diff
@@ -0,0 +1,28 @@
+package step3;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Car 테스트")
+class CarTest {
+    Car car;
```

Comment:
```suggestion
    private Car car;
```

굳이 명시하실 필요는 없지만, 접근 제한자를 추가해주시는건 어떨까요?